packages/Users/Models/User.php

```diff
@@ -121,4 +124,39 @@ public static function browse($options = [])
             'data'   => $find->get(),
         ];
     }
+
+    /**
+     * set status is block
+     * 
+     * @param  int $status
+     * @return int
+     */
+    public function block()
+    {
+        $this->status = $this->status | User::STATUS_BLOCK;
+        return $this->save();
+    }
+
+    /**
+     * set status is non block
+     * 
+     * @param  int $status
+     * @return int
+     */
+    public function unBlock()
+    {
+        $this->status = $this->status & ~User::STATUS_BLOCK;
+        return $this->save();
+    }
+
+    /**
+     * check status is block
+     * 
+     * @param  int  $status
+     * @return boolean
+     */
+    public function isBlock()
+    {
+        return ($this->status)==($this->status & User::STATUS_BLOCK);
```

Comment:
@gghoantt: anh nghĩ chỗ này chưa đúng đối với trường hợp status có lưu trữ trạng thái khác (ví dụ có thêm thông tin active mail).
Theo lý thuyết thì phải là `return (User::STATUS_BLOCK)==($this->status & User::STATUS_BLOCK);` chứ.
Em xem lại thử hì.